pallets/parachain-staking/src/lib.rs

```diff
@@ -2865,6 +2873,27 @@ pub mod pallet {
 				log::error!("💥 keeping old session because of empty collator set!");
 				None
 			} else {
+				// take snapshot of these collators' staking info
```

Comment:
Guess we'll need to refresh staking snapshot even if the collator set didn't change - Considering stake might have been modified. @sfffaaa what do you think?

Reply:
Yep, you are right. We have to retake the snapshot every session